tests/unit/test_cli.py

```diff
@@ -25,10 +25,18 @@
 
 @pytest.fixture
 def lifecycle_pack_mock():
-    """Mock for ui.init."""
+    """Mock for ui.pack."""
     patcher = patch("rockcraft.lifecycle.pack")
     yield patcher.start()
     patcher.stop()
+    
+
+@pytest.fixture
+def lifecycle_init_mock():
```

Comment:
you could use the mocker fixture for simplicity directly in the test method

Reply:
agreed. That's my common practice as well. However, this seems to be the strategy in all tests. 

I'd suggest going through all of these in a separate refactoring thread: https://github.com/canonical/rockcraft/issues/37